recipes/lsst_dashboard/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "lsst_dashboard" %}
+{% set version = "0.0.2a" %}
```

Comment:
Hi @kcpevey - we don't typically package alpha versions (due to CI limitations). Is there a full version you can package?

Reply:
We will be doing a beta release in about a week or two for folks at LSST to test. Would that qualify? or should we wait till the package is more mature?